cmd/faultinjector/main_test.go

```diff
@@ -236,6 +236,47 @@ func TestFaultInjector_VerbatimPassthrough(t *testing.T) {
 	require.NotZero(t, rawFrames)
 }
 
+func TestFaultInjector_ExcludePayloadData(t *testing.T) {
+	testEnv.SkipIfNotLive(t)
+
+	testData := mustCreateFaultInjector(t, newSlowTransferFrames, []string{"--delay", "1s", "--exclude-payload-data"})
+
+	{
+		sender, err := testData.ServiceBusClient.NewSender(testData.ServiceBusQueue, nil)
+		require.NoError(t, err)
+
+		for i := range 2 {
+			err = sender.SendMessage(context.Background(), &azservicebus.Message{
+				Body: []byte(fmt.Sprintf("hello world %d", i)),
+			}, nil)
+			require.NoError(t, err)
+		}
+
+		err = sender.Close(context.Background())
+		require.NoError(t, err)
+	}
+
+	// the default 10s delay is going to make it so we can't receive more than 1 message at a time.

```

Comment:
The test code here seems incorrect - it doesn't match what the test is actually testing.

I think it just needs a little cleanup (it looks like it came from one of the other tests that actually cares about transfer delays)